content/general/concepts/open-source/open-source.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: 'Open Source'
+Description: 'Open source refers to software that is released with the source code made freely available for modification and possible redistribution.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Conceptual'
+  - 'Development'
+  - 'Linux'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+The term "open source" refers to software that is freely available for modification and possible redistribution. Open source projects allow the source code to be reused by anyone with the restriction that any derivative projects are released under the same license.
+
+Such projects are intended to foster open collaboration from a variety of perspectives on problem-solving. The code is reviewed and updated by groups of developers of all sizes. One benefit of this is that it provides a greater opportunity for bugs and security holes to be detected and handled effectively.
+
+Open source software (OSS) is also thought to be more trustworthy. Since all the code is available to the public, any unwanted or nefarious behavior or activity will be quickly exposed by the development community surrounding the project.
+
+## Examples of OSS
+
+- Linux: Ever since the Linux kernel was released by [Linus Torvalds](https://www.codecademy.com/resources/docs/general/linus-torvalds) in 1991, an endless variety of Linux distributions have been produced, all similarly open source. The majority of software for the Linux operating system is open source as well.
```

Comment:
```suggestion
- [Linux](https://www.linux.org/): Ever since the Linux kernel was released by [Linus Torvalds](https://www.codecademy.com/resources/docs/general/linus-torvalds) in 1991, an endless variety of Linux distributions have been produced, all similarly open source. The majority of software for the Linux operating system is open source as well.
```

Reply:
@SSwiniarski Took the liberty of giving this a look-through! Some minor things